### ISSUE TITLE
check_ntp_offset - critical escalation

### ISSUE THREAD

**Issue by kafos** [ConSol-Monitoring/snclient#84]
I cannot understand, why I get a critical escalation here
![image](https://github.com/ConSol-Monitoring/snclient/assets/29919422/3b5f57ac-21aa-4201-9184-9eccc4eb5630)

The check config is default:
check_ntp_offset -a 'warn=offset > 50 || offset < -50' 'crit=offset > 100 || offset < -100'

snclient+ v0.16 ist running on windows 10.

w32tm / query /status on the windows 10 system:
![image](https://github.com/ConSol-Monitoring/snclient/assets/29919422/5fc8d113-5cf5-4737-a4ba-453cc8af4456)

**Comment by sni** [comment]
The parser assumes english locales when parsing the output.

**Comment by kafos** [comment]
Yes that makes sense - if it's possible to expand the check with german locale, that would be nice. 
Thanks

**Comment by sni** [comment]
next version will have a alternative line number based parser. Which works well for german so far.
